Assets/Scripts/CameraController.cs

```diff
@@ -14,35 +14,62 @@ public class CameraController : MonoBehaviour
     //used for rotation
     private float _originalY;
     private float _targetY;
-    private bool _isRotating;
     private float _speed = 380f;
+    private bool _isKeyboardRotating;
+    private bool _isMouseRotating;
     private float _movedAmount;
+    private float _mouseX;
+    private float _mouseY;
+    private float _mouseXSpeed = 1;
+    private float _mouseYSpeed = 0.1f;
+    private float _currentYDisplacement;
+    private const float MaxYDisplacement = 30;
+
 
     private void Start()
     {
         _offset = transform.position - PlayerObject.transform.position;
-        _offset.y = 0;
+        //_offset.y = 0;
```

Comment:
Could you remove the comments in this pr?

Reply:
well, i could, but we still didn't really decide on vertical camera movement. the comments are a quick half fix in case we decide to either do slow snap or lock y movement. should I still remove them anyway?